src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpGatewayApplicationViewModel.cs

```diff
@@ -8,12 +9,15 @@ public class RoatpGatewayApplicationViewModel : OrganisationDetailsViewModel
         public Guid Id { get; }
         public Guid ApplicationId { get; set; }
         public Guid OrgId { get; }
-        
+
         public string ApplicationStatus { get; }
-        public string GatewayReviewStatus { get; }
-        
+        public string GatewayReviewStatus { get; set; }
```

Comment:
why the get? ReviewStatus should be set within Apply Internal API

Reply:
It was get, but we need the set as we changing GatewayReviewStatus on first visit of any section.